migrate.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+
+repo_url="https://github.com/vsc-eco/vsc-deployment.git"
+default_path=$(realpath "../")
+
+if [ -z "$1" ]; then
+    echo "VSC node migration script. The default path is for the deployment repository is '${default_path}'"
+    read -p "Enter the path to clone the repository [${default_path}]: " clone_path
+    clone_path="${clone_path:-$default_path}"
+else
+    clone_path="$1"
+fi
+
+clone_path=$clone_path"/vsc-deployment"
+
+mkdir -p "$clone_path" || { echo "Failed to create directory at ${clone_path}"; exit 1; }
```

Comment:
Git will make the directory for you, right?
```suggestion
```

Reply:
indeed, just checked it, thought it wouldnt do so if you supply a path